src/dnsperf.c

```diff
@@ -381,6 +383,11 @@ print_statistics(const config_t* config, const times_t* times, stats_t* stats, u
         printf("  %s interrupted:  %" PRIu64 " (%.2lf%%)\n",
             units, stats->num_interrupted,
             PERF_SAFE_DIV(100.0 * stats->num_interrupted, stats->num_sent));
+    if (stats->num_unexpected > 0)
+        printf("  Unexpected IDs:       %" PRIu64 " (%.2lf%%)\n",
+            stats->num_unexpected,
+            PERF_SAFE_DIV(100.0 * stats->num_unexpected, stats->num_sent));
```

Comment:
It's questionable if % from num_sent if good choice, or if it could be percentage from num_timedout...